src/applications/vaos/services/appointment/index.js

```diff
@@ -374,3 +374,28 @@ export function isVideoVAFacility(appointment) {
 export function isVideoStoreForward(appointment) {
   return VIDEO_TYPES.storeForward === getVideoKind(appointment);
 }
+
+/**
+ * Method to check for the existence of a practitioner
+ * @param {Object} appointment An appointment resource
+ * @return {Boolean} Returns whether or not the appointment has a practitioner.
+ */
+export function hasPractitioner(appointment) {
+  const participant = appointment?.participant;
```

Comment:
I think you could collapse this into `return !!appointment?.participant?.some(item => item.actor?.reference?.includes('Practitioner'));`